.env.example

```diff
@@ -1,3 +1,4 @@
 REACT_APP_API_URL=http://localhost:3001
 REACT_APP_ALLOW_SIGNUP=true
-REACT_APP_MAINTENANCE_MODE=false
\ No newline at end of file
+REACT_APP_MAINTENANCE_MODE=false
+REACT_APP_HEAP_PRODUCTION_ID=111111111
```

Comment:
Why `PRODUCTION`? The environment itself will determine if it's production or not

Reply:
i thought i need to declare seperate id's for production and development